python/destinations/InfluxDB/library.json

```diff
@@ -0,0 +1,75 @@
+{
+  "libraryItemId": "",
```

Comment:
libraryItemId needs to be a valid, unique GUID.
You can generate one using https://guidgenerator.com/online-guid-generator.aspx
Same issue with the source connector